resources/spec.yaml

```diff
@@ -10631,6 +10641,17 @@ definitions:
           - dest: userId
             src: userId
         operationId: updateOrgContactUser
+    x-okta-crud:
+      - alias: read
+        arguments:
+          - dest: contactType
+            src: id
+        operationId: getOrgContactUser
+      - alias: update
+        arguments:
+          - dest: orgContactUser
+            self: true
+        operationId: updateOrgContactUser
```

Comment:
this operation could not be located in a CRUD, because the `updateOrgContactUser` operation require the `UserIdString` but not the `OrgContactTypeObj`. Also this operation already defined in `x-okta-operations`.